### PR TITLE
Business plan added to docs

### PR DESCRIPTION
**Pull request by bgrenon**
Ready for review:
- [plans](https://neon-next-git-bgrenon-business-plan-neondatabase.vercel.app/docs/introduction/plans#business)
- [support](https://neon-next-git-bgrenon-business-plan-neondatabase.vercel.app/docs/introduction/support)

FYI, many small updates throughout the docset.

### CODE REVIEW

File: content/docs/introduction/extra-usage.md [hunk 1]
```diff
@@ -70,15 +71,15 @@ Taking advantage of extra usage requires no user action. Extra usage, if support
...
+For example, the Launch plan includes an allowance of 10 GiB in the plan's monthly fee. If you exceed 10 GiB of storage at any point during the month, you are automatically allocated an extra storage unit of 2 GiB at $3.50 per unit. If you exceed 12 GiB, you are allocated 2 units of 2 GiB (an extra $7), and so on. It works the same way on the Scale and Business plans, but with 10 GiB units of storage at $15 per unit. However, the extra charge is prorated from the date the extra usage was allocated, meaning that you are not billed the full amount if extra storage units were allocated partway through the month.
```
Comment (bgrenon) [inline content/docs/introduction/extra-usage.md:74]:
```suggestion
For example, the Launch plan includes an allowance of 10 GiB in the plan's monthly fee. If you exceed 10 GiB of storage at any point during the month, you are automatically allocated an extra storage unit of 2 GiB at $3.50 per unit. If you exceed 12 GiB, you are allocated 2 units of 2 GiB (an extra $7), and so on. It works the same way on the Scale and Business plans, but with 10 GiB units of storage at $15 per unit for Scale and $7.50 per unit for Business. However, the extra charge is prorated from the date the extra usage was allocated, meaning that you are not billed the full amount if extra storage units were allocated partway through the month.
```

**Comment by bvanvugt** [comment]
@bgrenon we should probably separate them, tbh 😞  the scale updates might not happen next week.

File: content/docs/introduction/support.md [hunk 2]
```diff
@@ -37,29 +55,127 @@ You can open support tickets in the Neon Console. Look for the **Support** link
...
+
+A detailed explanation of each severity level, including several examples, is provided below.
+
+### Severity 1 (Critical)
+
+- Catastrophic problems in the Customer’s production system leading to loss of service or impact on the Customer’s business
+- Unavailability of the service
+- Security breaches that compromise the confidentiality, integrity, or availability of the database or its data.
+
+**Note**: If Critical is selected during the case creation, the customer will be asked to provide in-depth details on the business impact the issue has caused.
```
Comment (danieltprice) [inline content/docs/introduction/support.md:102]:
```suggestion
<Admonition type="note">
If Critical is selected during the case creation, the customer will be asked to provide in-depth details on the business impact the issue has caused.
</Admonition>
```

File: content/docs/introduction/support.md [hunk 3]
```diff
@@ -37,29 +55,127 @@ You can open support tickets in the Neon Console. Look for the **Support** link
 
 You can expect an initial response time of 2 business days, from 6am to 6pm Pacific Standard Time (UTC -8), Monday through Friday, excluding public holidays in the United States. For custom support solutions, please contact [Sales](https://neon.tech/contact-sales).
 
-## Priority support
+### Priority support
 
-Neon's [Scale plan](/docs/introduction/plans#scale) includes **Priority** support.
+Neon's [Scale](/docs/introduction/plans#scale) and [Business](/docs/introduction/plans#business) plans includes **Priority** support.
```
Comment (MihaiBojin) [inline content/docs/introduction/support.md:60]: Scale is now Standard.

**Review by MihaiBojin — COMMENTED** [review]
<sub>_[Reviewed with Forge](https://codeium.com/forge/neondatabase/website/2082/review)_</sub>

**Comment by MihaiBojin** [comment]
Please also capture all changes described by comments in https://docs.google.com/document/d/13295hBdVOk8tOjvKgkoeyja-pA_tC1Mh9ve0sTGl484 (cc'd @bgrenon)

File: content/docs/introduction/support.md (hunk 3 above)
Comment (MihaiBojin) [reply content/docs/introduction/support.md:63]: @bgrenon ... plans include (not includes) :)

File: content/docs/introduction/support.md [hunk 4]
```diff
@@ -35,31 +56,129 @@ You can open support tickets in the Neon Console. Look for the **Support** link
...
+- Non-essential modifications to configuration, like adjusting database parameters or table schema
+- Minor performance concerns that have minimal impact on database usability
+- Minor issues related to application integrations, such as minor API connectivity problems
+- Small-scale challenges with data import/export, data transformation, or data loading processes
+
+Examples:
+
+1. Partial outage of the service provided by Neon: service usable, but key feature unusable, e.g.:
+   - Cannot create a new branch
+   - Cannot PITR
```
Comment (MihaiBojin) [inline content/docs/introduction/support.md:129]: Cannot execute a [branch restore](https://neon.tech/docs/guides/branch-restore).

File: content/docs/introduction/support.md [hunk 5]
```diff
@@ -35,31 +56,129 @@ You can open support tickets in the Neon Console. Look for the **Support** link
...
-| :----------- | :----------------------------------------------------- |
-| Community    | [Free Plan](/docs/introduction/plans#free-plan)        |
-| Standard     | [Launch plan](/docs/introduction/plans#launch)         |
-| Priority     | [Scale plan](/docs/introduction/plans#scale)           |
-| Enterprise   | [Enterprise plan](/docs/introduction/plans#enterprise) |
+| Support plan | Pricing plan                                                                                         |
+| :----------- | :--------------------------------------------------------------------------------------------------- |
+| Community    | [Free Plan](/docs/introduction/plans#free-plan)                                                      |
+| Standard     | [Launch](/docs/introduction/plans#launch)                                                            |
+| Priority     | [Scale](/docs/introduction/plans#scale) and [Business plan](/docs/introduction/plans#business) plans |
```
Comment (MihaiBojin) [inline content/docs/introduction/support.md:171]: same point about Scale moving to Standard

File: content/docs/introduction/support.md (hunk 3 above)
Comment (bgrenon) [reply content/docs/introduction/support.md:63]: Updated on support page and [plans](https://neon-next-git-bgrenon-business-plan-neondatabase.vercel.app/docs/introduction/plans) page.

**Review by danieltprice — APPROVED** [review]

**Pull request merged**